CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Ewon Flexy Cumulocity Connector Changelog
 
+## Version 1.4.2
+### Features
+- Update deployment dependencies
+### Bug Fixes
+- Fixed an issue where the main thread ends, but the event handler thread continues to run
+- Removes unnecessary backtick from tasks.json 
```

Comment:
Is this still in there? I didnt see it.

Reply:
Looks like it did get removed so this line should be removed.

Reply:
good catch. Removed now